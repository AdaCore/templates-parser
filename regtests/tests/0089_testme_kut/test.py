from test_support import *

run('testme', ["testme10.tmplt", "kut"])
run('print_tree', ["testme10.tmplt"])
