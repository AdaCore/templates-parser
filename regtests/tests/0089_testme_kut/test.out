
One 1

Unknown @_UPPER:unknown_@

  1

  ok, not EXIST

  ok, not EMPTY

  ok, @_unknown_@ not equal to 1


one @_unknown_@
two @_unknown_@
three @_unknown_@

[INFO] testme10.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[TEXT] One @_ONE_@
[TEXT] 
[TEXT] Unknown @_UPPER:unknown_@
[TEXT] 
[IF] (@_ONE_@ = 1)
  [TEXT]   1
[ELSE]
  [TEXT]   error
[END_IF]
[TEXT] 
[IF] @_EXIST:unknown_@
  [TEXT]   error
[ELSE]
  [TEXT]   ok, not EXIST
[END_IF]
[TEXT] 
[IF] @_IS_EMPTY:unknown_@
  [TEXT]   error
[ELSE]
  [TEXT]   ok, not EMPTY
[END_IF]
[TEXT] 
[IF] (@_unknown_@ = 1)
  [TEXT]   error, unknown = 1
[ELSE]
  [TEXT]   ok, @_unknown_@ not equal to 1
[END_IF]
[TEXT] 
[TEXT] 
[TABLE]
  [BLOCK]
    [COMMON]
      [TEXT] @_VAR3_@ @_unknown_@
  [END_BLOCK]
[END_TABLE]
